tools/cldr-code/src/main/java/org/unicode/cldr/tool/GeneratePersonNameTestData.java

```diff
@@ -69,7 +71,9 @@ public static void main(String[] args) {
             }
 
             try {
-                CLDRFile cldrFile = factory.make(locale, true);
+                CLDRFile cldrFile =
+                        factory.make(locale, true, DraftStatus.contributed); // don't include
+                // draft=unconfirmed/provisional
```

Comment:
I don't know what I'm looking at, but is `DraftStatus.contributed` a property of the whole file?  What I was seeing in some files was that _some_ person-name resources had their draft status set to "contributed" (or, actually, not mentioned at all), and _others_ had their draft status set to "provisional" or "unconfirmed".  Does this catch that?

Reply:
It sets the minimum bar for items to be included. So contributed means either contributed or approved (which is implicit if there is no draft value). That's the general bar we use for ICU, so this should get better results. cc @pedberg-icu